en/docs/wip/api-recommendations.md

```diff
@@ -0,0 +1,92 @@
+
+Using AI technologies, this feature will analyze the behavior of the developer, application that he or she is 
+developing and suggest a list of new APIs that are not subscribed by the user but might be useful for the project. 
+In order to recommend effectively, there must be at least 20 APIs in the portal.
+
+![](../assets/img/api-recommendations.png)
+
+## Solution
+Recommendation server will be deployed in WSO2 and will be provided to all the customers as a service. This service 
+is OAuth2 protected and if you want to use this feature in your developer portal, APIM team will provide access to 
+the recommendations API. Once the recommendations feature is activated, recommendation feature related events will be 
+published to this recommendations API. When a user logs in to the developer portal, recommended APIs based on that 
+portal will be sent from the service and that APIs will be listed.
+
+## To API Publishers
+If you want your API to be discovered by a larger audience, try to be descriptive when adding details of the API. 
+Recommendation features analyze the API name, API description, context, resource path, resource descriptions, resource 
+summary, tags and suggest the API in the developer portal. Meaningful descriptions with correct terminology will be 
+picked first and recommended to the user.
+
+## To API Subscribers
+Recommendation feature will analyze the Application name, Application details when recommending APIs. Adding a 
+detailed description about your application, it’s capabilities and the use case will be matched with the APIs that 
+are currently available in the Developer portal. Also the feature will analyze the APIs that the user has previously 
+viewed and the clicked tags when recommending an API. 
+
+### How to enable the feature
+
+Add the following configurations to the `<API-M_HOME>/repository/conf/deployment.toml` file and restart the server.
+
+```
+[apim.devportal.recommendations]
+recommendations_api = "<Recommendations service provided by WSO2>"
+consumer_key = "<Provided by the WSO2 APIM team>"
+consumer_secret = "<Provided by the WSO2 APIM team>"
+```
+
+### Additional configurations
+Only add the following configurations if you need to change the default behaviour.
+```
+max_recommendations = 10
+wait_duration = 10
+apply_for_all_tenants = false
+```
+
+- **max_recommendations**: Max recommendations displayed in the developer portal. (By default 6 APIs are displayed in
+ the developer portal)
+- **wait_duration**: Recommendations for users are requested from the API in 15 minutes intervals. If that interval
+ needs to be changed, add the time in minutes.
+- **apply_for_all_tenants**: In a multi tenant deployment, if recommendations feature is enabled, it will show
+ recommendations in all the developer portals and if you need to apply the recommendation feature only for selected 
+ tenants, add false here. 
+
+### Enable recommendations feature for a specific tenant
```

Comment:
shall we reword it to **only for specific tenants**? I assume it will be enabled to all tenants if it is enabled in the deployment.toml.

Reply:
Fixed